### PR TITLE
Add last year daily data

### PR DESCRIPTION
**Pull request by cptCloud**
The intent is to compare it with current year daily data.
The idea came after I did some renovations and wanted to compare
consumption usage before and after.

Next step would be to add it to home assistant.

### CODE REVIEW

**Comment by cptCloud** [comment]
Hi @titilambert  
I added the support for this PR in Home assitant. What do you think?

**Comment by titilambert** [comment]
@cptCloud thanks !
For Home assitant, I'm not sure that we should continue to support the component because the component is just parsing the HydroQuebec website and Home-Assistant is trying to remove this kind of compontent.
That's why, I'm thinking creating a small daemon which will send data to Home-Assistant through mqtt

**Comment by cptCloud** [comment]
Did you take a chance and asked Hydroquebec to develop an API or even a HA component now it is so popular?
Anyways, a mqtt deamon sounds good and will perfectly fit in a container.

File: pyhydroquebec/client.py [hunk 1]
```diff
@@ -447,6 +451,7 @@ def fetch_data(self):
             # At the end/starts of a period
             if daily_data:
                 daily_data = daily_data[0]['courant']
+                daily_data_compare = daily_data[0]['compare']
```
[the remark this answers is not in the document]
Comment (cptCloud) [reply pyhydroquebec/client.py]: Yes you are right. I moved it for some reason.
Moved it back at 453 and it is working locally.

**Comment by titilambert** [comment]
This should be fixed in the 3.x version. The compare data are now collected. The only part missing is the output

**Comment by starwolf73** [comment]
> > Did you take a chance and asked Hydroquebec to develop an API or even a HA component now it is so popular?
> 
> No :( I don't know anybody at HydroQuebec :/ And they never answer to my tweet (https://twitter.com/ttb_lt/status/1068863516206084096)
> 
> > Anyways, a mqtt deamon sounds good and will perfectly fit in a container.
> 
> Perfect, I'm starting to rewrit of this library to simplify the maintenance.
> I will push it to a new branch tonight

Just for your information, I contact HydroQuebec last week and ask if they can provide any information for their public API. They reply back that IT team is looking into it and will contact in short time. Will let you know if they provide any input.

**Comment by titilambert** [comment]
@starwolf73 THAT'S SO COOL ! thanks a lot !
Do you think they will really reply ?
Could you open an issue about this ? Just to be able to follow the news about it
Thanks again !

**Pull request closed without merging**